docs/phylum.md

```diff
@@ -0,0 +1,37 @@
+---
+title: phylum
+category: 6255e67693d5200013b1fa3e
+hidden: false
+---
+
+Interact with the Phylum API.
+
+### Core Commands
+* [phylum analyze](https://docs.phylum.io/docs/phylum_analyze)
+* [phylum auth](https://docs.phylum.io/docs/phylum_auth)
+* [phylum project](https://docs.phylum.io/docs/phylum_project)
+
+### Additional Commands
+* [phylum help](https://docs.phylum.io/docs/phylum_help)
+* [phylum history](https://docs.phylum.io/docs/phylum_history)
+* [phylum package](https://docs.phylum.io/docs/phylum_package)
+* [phylum ping](https://docs.phylum.io/docs/phylum_ping)
+* [phylum update](https://docs.phylum.io/docs/phylum_update)
+* [phylum version](https://docs.phylum.io/docs/phylum_version)
+
+### Options
+  `-c`, `--config <FILE>`
+  &emsp; Use a specific configuration file
```

Comment:
These options mostly match the command line help output. Should they match? Can we generate argument documentation from a common source to keep them in sync?

Reply:
You aren't wrong. It would be ideal to sync these things up from a common source, we should investigate how we might do that in the future.